sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/Argument.cs

```diff
@@ -146,10 +146,7 @@ public static void AssertNotClosed(bool wasClosed, string targetName)
         {
             if (wasClosed)
             {
-                throw new ServiceBusException(
-                    string.Format(CultureInfo.CurrentCulture, Resources.ClosedInstanceCannotPerformOperation, targetName),
-                    ServiceBusException.FailureReason.ClientClosed,
-                    targetName);
+                throw new ObjectDisposedException(targetName, string.Format(CultureInfo.CurrentCulture, Resources.ClosedInstanceCannotPerformOperation, targetName));
```

Comment:
Do we still need AssertNotClosed or can we use AssertNotDisposed?

Reply:
We did not need it anymore! I have updated it to be removed and used AssertNotDisposed instead. 😊